### DISCUSSION TITLE
Open a web window in the space

### DISCUSSION THREAD

**Discussion by fenrir-liutiecheng** [mozilla/hubs#3204] [Hubs Cloud Questions] [answered]
I want to click or place the mouse on an object in the space, a web window will pop up in the space, and a certain webpage can be displayed in the window. How can I do this?

**Comment by robinkwilson** [comment, marked as answer]
I'm unable to give specific instructions to implement that kind of feature. The community has been asking for triggerable objects for a while now (click and something happens). So keep an eye out for that functionality in the future!

I recommend visiting the Hubs community discord to see if any community members would like to collaborate to make this or point you in the right direction. 

Will leave this ticket open so others can pipe in.

**Reply by aelainelong** [reply]
@robinkwilson While opening a web window may not yet be possible, it is currently possible to display a custom `div` or other HTML elements on object click, correct? I think I have seen this done before in Hubs Cloud environments using custom JavaScript.

**Comment by cvanmeurs** [comment]
I brought this up a while ago too, and was told that there are security concerns when it comes to displaying a web page inside of a Hubs room.  I have been told that it's technically possible to do, but it's not a feature that we are considering at the moment.

The only thing you can currently do is place a link within your Hubs room that points to an external website.  Or you could screenshare a monitor that has the website open, but it would not be interactive.
